library/src/test/java/io/karn/notify/NotifyMetaTest.kt

```diff
@@ -60,7 +61,7 @@ class NotifyMetaTest : NotifyTestBase() {
         Assert.assertEquals(testCancelOnClick, (notification.flags and NotificationCompat.FLAG_AUTO_CANCEL) != 0)
         Assert.assertEquals(testCategory, notification.category)
         Assert.assertEquals(testTimeout, notification.timeoutAfter)
-        Assert.assertEquals(1, notification.extras.getStringArray(NotificationCompat.EXTRA_PEOPLE)?.size
+        Assert.assertEquals(1, notification.extras.getStringArrayList(Notification.EXTRA_PEOPLE_LIST)?.size
```

Comment:
Note for later: This is the bulk of the change -- seems like API 28 has made some changes to the way that the `people` attribute is stored within the notification's `extras` Bundle.